### PR TITLE
API-914: Add helper message when no connections exist

### PR DESCRIPTION
**Pull request by pchasle**
**Description (for Contributor and Core Developer)**

Without permission
image

With permission
image

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Apps/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml [hunk 1]
```diff
@@ -127,3 +127,12 @@ akeneo_apps:
...
                 today: Today
+        no_app:
+            title: No connection declared yet.
+            message_with_permission:
+                link: Create one here!
+                link_url: https://www.akeneo.com/
+            message_without_permission:
+                message: Ask your administrator to create one.
+                link: Learn more about connections here...
+                link_url: https://www.akeneo.com/
```
Comment (willy-ahva) [inline src/Akeneo/Apps/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml]: blank line

File: src/Akeneo/Apps/front/src/application/dashboard/components/NoApp.tsx [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+            </SecurityGuard>
+        </PageError>
+    );
+};
 
-const Title = styled.div`
-    height: 36px;
-    font-size: ${({theme}: PropsWithTheme) => theme.fontSize.title};
-    color: ${({theme}: PropsWithTheme) => theme.color.grey140};
+const Link = styled.a`
```
Comment (willy-ahva) [inline src/Akeneo/Apps/front/src/application/dashboard/components/NoApp.tsx]: seems to be duplicated. Should we move it in /common or /shared ?

File: src/Akeneo/Apps/front/src/application/shared/security/SecurityGuard.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React, {PropsWithChildren, ReactNode, FC} from 'react';
+import {useSecurity} from './use-security';
+
+type Props = PropsWithChildren<{
+    acl: string;
+    fallback?: ReactNode;
```
Comment (willy-ahva) [inline src/Akeneo/Apps/front/src/application/shared/security/SecurityGuard.tsx]: good idea :+1:

File: src/Akeneo/Apps/front/src/application/dashboard/components/NoApp.tsx (hunk 2 above)
Comment (pchasle) [reply src/Akeneo/Apps/front/src/application/dashboard/components/NoApp.tsx]: I thought of this too. But didn't want to clutter too much the _common_ folder. Maybe it's fine for now and if we have more instances of this type of link, extract it.

**Pull request merged**